libs/designsystem/src/lib/components/modal/footer/modal-footer.component.scss

```
@@ -7,4 +7,6 @@ ion-footer {
   justify-content: center;
   align-items: center;
   background-color: get-color('white');
+  padding-bottom: calc(#{size('s')} + var(--keyboard-offset, 0px));
+  transition: padding-bottom 250ms ease-out;
```

Consider using `get-transition-easing` function:
```suggestion
  transition: padding-bottom 250ms #{get-transition-easing('enter')}
```